packages/components/src/index.js

```diff
@@ -43,6 +44,9 @@ export { default as DropZoneProvider } from './drop-zone/provider';
 export { default as Dropdown } from './dropdown';
 export { default as DropdownMenu } from './dropdown-menu';
 export { default as ExternalLink } from './external-link';
+export { default as __experimentalFlex } from './flex';
+export { default as __experimentalFlexBlock } from './flex/block';
+export { default as __experimentalFlexItem } from './flex/item';
```

Comment:
Should we expose these or just keep them internal for now.

Reply:
Good question!

Across my several branches, I haven't needed to use these externally yet.
However, I think components/blocks can benefit from the `Flex` component.

Are there any hesitations to exposing them publicly as experimental?

(Thank you! ❤️)

Reply:
My hesitation comes from two perspectives:

 - I already know CSS flexbox, why I should learn how to new components and their props. Is this abstraction simpler? what problems it auto-solve?
 - We've been too conservative with __experimental APIs, when do you think we should make these stable? Are we doing so for previous things we marked as experimental?

Reply:
Thank you for your perspective! I've added additional content to the Flex README.md file to (hopefully) demonstrate how it may be helpful in rendering layouts.

I've also removed the `__experimental` prefix for the Flex components.